src/cbor/CBORDecoder.h

```diff
@@ -23,6 +23,7 @@
  ******************************************************************************/
 #ifdef __AVR__
 #include <Arduino.h>
+#include <avr_stl.h>
```

Comment:
@facchinm The "Compile Examples" workflow is configured to install this library: https://github.com/mike-matera/ArduinoSTL
but it looks like maybe you wanted this one instead: https://github.com/ciband/avr_stl

I'm happy to update the workflow if needed, or you can just change this line:
https://github.com/arduino-libraries/ArduinoIoTCloud/blob/484cca9d4bcfa99a819a9f4dc5e2cc3241feb536/.github/workflows/compile-examples.yml#L82
to:
```yaml
              - name: avr_stl
```

Reply:
The last one seems more actively maintained and I submitted the PR to fix compilation there (https://github.com/ciband/avr_stl/pull/6). So, in theory, to make CI happy we could use my fork now :smile:

Reply:
>we could use my fork now

Would you like me to submit a PR to this `offload-ssl-nina-take2` branch for that modification to the CI workflow?

Reply:
> The last one seems more actively maintained and I submitted the PR to fix compilation there ([ciband/avr_stl#6](https://github.com/ciband/avr_stl/pull/6)). So, in theory, to make CI happy we could use my fork now

Sorry to be the crashing the party here but satisfying CI is not top priority here. Top priority is getting a stable code base working for the largest amount of users without needing to install forked libraries in private repositories.

Reply:
CI is just the tool telling us when we deviate too much from easy usability because we need to bend-over backwards to fulfil a given task.

Reply:
Sure, but the whole stuff is going to be compiled primarily in the cloud, so the need to install forked libraries would not affect the user.

Reply:
I'm not sure that's the case and even so - pre-installing an invisible library for all cloud users is setting us up for an avoidable mistake in the future. Invisible dependencies are sure to be forgotten.